docs: add information about global config options (#3466)
SumoLogic/sumologic-kubernetes-collection: docs/best-practices.md

disable collection of this data in order to lower the ingest.

In order to learn more about filtering out data please see [the doc about filtering data](/docs/filtering.md).

## Global customizations

To retain some configuration options across the Helm Chart, it's recommended to use global configuration keys. Below are listed options that
can be set globally for the main chart and the subcharts.

> **Note:** The subcharts also have more configurations underneath. To make sure you don't miss anything you want to change, make sure to
> check respective documentation. Links to documentations of subcharts can be found [here][subcharts-docs].

Reviewer: ```suggestion
**Note:** Each subchart can be independently configured. To make sure you don't miss anything you want to change, make sure to
check their documentation. Links to documentations of subcharts can be found [here][subcharts-docs].
```

| component                | key                                                |
| ------------------------ | -------------------------------------------------- |
| `remoteWriteProxy`       | `sumologic.metrics.remoteWriteProxy.nodeSelector`  |
| `otelcolInstrumentation` | `otelcolInstrumentation.statefulset.nodeSelector`  |
| `tracesGateway`          | `tracesGateway.deployment.nodeSelector`            |
| `otellogs`               | `otellogs.daemonset.nodeSelector`                  |
| `setupJob`               | `sumologic.setup.job.nodeSelector`                 |
| `tracesSampler`          | `tracesSampler.deployment.nodeSelector`            |
| `metadata`               | `metadata.metrics.statefulset.nodeSelector`        |
| `metadata`               | `metadata.logs.statefulset.nodeSelector`           |
| `pvcCleaner`             | `pvcCleaner.job.nodeSelector`                      |
| `metrics otelcol`        | `sumologic.metrics.collector.otelcol.nodeSelector` |

[k8s-node-selector]: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector

### Tolerations

> These options allow you to set node selectors for whole subcharts. The values under these keys must be a list of tolerations. [More
> information about node selectors can be found here][k8s-tolerations].

Reviewer: ```suggestion
These options allow you to set tolerations for whole subcharts. The values under these keys must be a list of tolerations. [More
information about node selectors can be found here][k8s-tolerations].
```
